### PR TITLE
apps: move pkg/apps/cmd to pkg/oc/cli/deploymentconfigs

### PR DESCRIPTION
**Pull request by mfojtik**
Because `pkg/apps/cmd` is not used by `pkg/apps` and it is only used by `oc` (scaler, history, rollback). It really belongs to `oc` package.

### CODE REVIEW

**Comment by tnozicka** [comment]
/hold
I think is wrong, I don't see what does all the command like `oc history` or `oc delete dc/` have to do with deployer

**Comment by tnozicka** [comment]
@mfojtik can we call it `pkg/oc/cli/deploymentconfigs` to avoid ambiguity with upstream Deployments? (or call it apps in general)

File: pkg/cmd/infra/deployer/doc.go [hunk 1]
```diff
@@ -1,3 +1,3 @@
 // Package cmd contains various interface implementations for command-line tools
 // associated with deploymentconfigs.
-package cmd
+package deployer
```
Comment (tnozicka) [inline pkg/cmd/infra/deployer/doc.go]: why is it moving to deployer? is that just a leftover from the change?

Comment (mfojtik) [reply pkg/cmd/infra/deployer/doc.go]: that is goland not being that smart as i thought :-) good catch

**Comment by mfojtik** [comment]
> @mfojtik can we call it pkg/oc/cli/deploymentconfigs to avoid ambiguity with upstream Deployments? (or call it apps in general)

`apps` is by me even more confusing in context of `pkg/oc/cli/apps` :-) We can call it 'rollouts' or something. I don't feel strong on names here as long as we pick one we can stick to for ~18 months ;-))

**Comment by tnozicka** [comment]
apps are not the best name, I agree

If there won't be anything else there, and we can likely just create separate directory in that case, my vote is for `pkg/oc/cli/deploymentconfigs` as all those files are hooks into oc to handle cases of commands for DeploymentConfigs and DC_deployments which are a subset of DC

**Comment by mfojtik** [comment]
@tnozicka deploymentconfigs sounds ok

**Pull request merged**